.storybook/main.ts

```diff
@@ -0,0 +1,25 @@
+import { TsconfigPathsPlugin } from 'tsconfig-paths-webpack-plugin';
+import type { StorybookConfig } from '@storybook/react-webpack5';
+
+const config: StorybookConfig = {
+  stories: ['../src/**/*.mdx', '../src/**/*.stories.@(js|jsx|ts|tsx)'],
+  addons: [
```

Comment:
We had a postcss addon, but it's been deprecated. We weren't using it so I dropped it.
Theming addon was not migrated to 7, I've replaced it with a custom decorator and a toolbar